web.go

```diff
@@ -25,12 +25,13 @@ var metrics = stats.New()
 // WebProvider is a provider.Provider implementation that provides the UI.
 // FIXME to be handled another way.
 type WebProvider struct {
-	Address  string `description:"Web administration port"`
-	CertFile string `description:"SSL certificate"`
-	KeyFile  string `description:"SSL certificate"`
-	ReadOnly bool   `description:"Enable read only API"`
-	server   *Server
-	Auth     *types.Auth
+	Address    string `description:"Web administration port"`
+	CertFile   string `description:"SSL certificate"`
+	KeyFile    string `description:"SSL certificate"`
+	ReadOnly   bool   `description:"Enable read only API"`
+	Statistics bool   `description:"Enable more detailed statistics"`
```

Comment:
Maybe we could use instead:

```
type WebProvider struct {
	Address    string `description:"Web administration port"`
 	CertFile string `description:"SSL certificate"`		 
 	KeyFile  string `description:"SSL certificate"`		 
 	ReadOnly bool   `description:"Enable read only API"`		
	Statistics *types.Statistics   `description:"Enable more detailed statistics"`
	...
}
```

and in `types.go`:
```
type Statistics struct{
 	RecentErrors int  `description:"Recent errors displayed"`		
}
```

With a default value set to 10. Statistics would still be enable using `--web.statistics` :)

WDYT?